install.sh

```diff
@@ -134,12 +123,12 @@ else
 				echo "Indigo, the SubjuGator repository at an earlier date,  and all of the old"
 				echo "SubjuGator dependencies will need to be downloaded and installed."
 				echo -n "Do you still wish to proceed? [y/N] " && read RESPONSE
-				if ([ "$RESPONSE" = "Y" ] || [ "$RESPONSE" = "y" ]); then
+				if ([ "$RESPONSE" == "Y" ] || [ "$RESPONSE" == "y" ]); then
```

Comment:
The brackets are what defines a conditional statement in bash, not the operator. if you run `which [`, you will see that `/usr/bin/[` is actually a program belonging to the test package. This is part of POSIX shell and only supports `=` for a conditional check. In modern bash, this has been replaced with a `[[` built-in that supports both `==` and `=` for compatibility with POSIX. Realistically, I should be using `[[`, but bash allows me to shorthand it to `[`.